include/HubFramework/HUBComponentWithChildren.h

```diff
@@ -84,11 +84,13 @@ NS_ASSUME_NONNULL_BEGIN
  *
  *  @param component The parent component
  *  @param childIndex The index of the child component that was selected
+ *  @param customData Any custom data to send to the action. Will be available on the
```

Comment:
The rest of the API doesn't mention anything about actions. So this documentation is a bit out of context. How about doing something like:

```
Any custom data to use when the selection is handled. Will be available on the `HUBActionContext` passed to any actions handling the selection.
```